### PR TITLE
Attempt to ensure we are using python3 for amalgamation

### PR DESCRIPTION
**Pull request by bam241**
This attempt to enforce python3 usage when amalgamating PyNE

### CODE REVIEW

File: doc/CHANGELOG.rst [hunk 1]
```diff
@@ -13,6 +13,7 @@ Next version
    * Including installation of a CMake version file for use with `find_package` in client codes. (#722)
    * CMake option to checkout PyNE submodule automatically (#734)
    * GitHub Action to build and upload Docker images. (#746, #748, #754, #757, #758, #759, #765, #767)
+   * Ensuring Python3 is present and enforcing it for PyNE amalgamation
```
Comment (gonuke) [inline doc/CHANGELOG.rst]: please
```suggestion
   * Ensuring Python3 is present and enforcing it for PyNE amalgamation (#773)
```

**Comment by bam241** [comment]
sadly the find_package method did not work as it requires cmake 3.12 min.... I was not ready to bump our cmake requirement....

**Comment by gonuke** [comment]
This passes tests... does it do what you need?

**Comment by bam241** [comment]
I need to check in Cubit-plugin where we have problem with pyne amalgamation, I'll ping you when I know more

**Comment by bam241** [comment]
@gonuke it worked as intended see: svalinn/Cubit-plugin#107

I am happy with those changes

**Comment by gonuke** [comment]
Thanks @bam241

**Pull request merged**